test/benchmarks/material/table/BUILD.bazel

```diff
@@ -8,6 +8,10 @@ component_benchmark(
         "@npm//protractor",
         "@npm//@types/jasmine",
     ],
+    ng_assets = [
+        ":app.module.html",
```

Comment:
We should probably rename this file to be based on the component it is used within? Seems confusing that it's named `app.module.html` as modules don't have any templates.

Reply:
I've been just writing the component and module in the same file for the perf benchmarks 😬 In retrospect that might not have been a great idea haha